api/v1beta1/disruption_webhook_test.go

```diff
@@ -532,6 +533,19 @@ var _ = Describe("Disruption Webhook", func() {
 					Expect(ddmarkMock.AssertNumberOfCalls(GinkgoT(), "ValidateStructMultierror", 1)).To(BeTrue())
 				})
 			})
+
+			When("controller has network-disruptions disabled", func() {
+				It("should return an error which denies the creation of a new disruption", func() {
+					ddmarkMock.EXPECT().ValidateStructMultierror(mock.Anything, mock.Anything).Return(&multierror.Error{})
```

Comment:
```suggestion
					ddmarkMock.EXPECT().ValidateStructMultierror(mock.Anything, mock.Anything).Return(&multierror.Error{}).Once()
```

nit: then you can remove the line 546 😉